apps/token/src/routes/governance/components/proposals-list-item/proposals-list-item-details.spec.tsx

```diff
@@ -0,0 +1,370 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import { AppStateProvider } from '../../../../contexts/app-state/app-state-provider';
+import { VegaWalletContext } from '@vegaprotocol/wallet';
+import { MockedProvider } from '@apollo/client/testing';
+import { render, screen } from '@testing-library/react';
+import { format } from 'date-fns';
+import {
+  ProposalRejectionReason,
+  ProposalState,
+  VoteValue,
+} from '@vegaprotocol/types';
+import {
+  generateNoVotes,
+  generateProposal,
+  generateYesVotes,
+} from '../../test-helpers/generate-proposals';
+import { ProposalsListItemDetails } from './proposals-list-item-details';
+import { DATE_FORMAT_DETAILED } from '../../../../lib/date-formats';
+import {
+  mockPubkey,
+  mockWalletContext,
+  networkParamsQueryMock,
+  amendToDate,
+  lastWeek,
+  nextWeek,
+} from '../../test-helpers/mocks';
+import type { Proposals_proposals } from '../../proposals/__generated__/Proposals';
+
+const oneDay = 60 * 60 * 24;
+const oneHour = 60 * 60;
+const oneMinute = 60;
+
+const renderComponent = (
+  proposal: Proposals_proposals,
+  mock = networkParamsQueryMock
+) => (
+  <Router>
+    <MockedProvider mocks={[mock]}>
+      <AppStateProvider>
+        <VegaWalletContext.Provider value={mockWalletContext}>
+          <ProposalsListItemDetails proposal={proposal} />
+        </VegaWalletContext.Provider>
+      </AppStateProvider>
+    </MockedProvider>
+  </Router>
+);
+
+it('Enacted', () => {
```

Comment:
chore (non-blocking): here, too, a describe wrapper would improve things, but feel free to ignore me, not super important